prow/cmd/branchprotector/protect.go

```diff
@@ -132,7 +133,7 @@ func main() {
 		for i, err := range errors {
 			logrus.WithError(err).Error(i)
 		}
-		logrus.Fatalf("Encountered %d errors protecting branches", n)
+		logrus.Errorf("Encountered %d errors protecting branches", n)
```

Comment:
I don't think this is a correct thing to do. Branchprotector is not a daemon, it executes and terminates. Exiting with non-zero is the right thing to do when something fails.

Reply:
I see and guess I didn't read it too well. In that case we can leave the Fatalf for sure.

Reply:
Reverted to Fatalf